Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -2839,7 +2839,8 @@ void MainWindow::processTextMessage(const MyTextMessage& textmsg)
             if(dlg)
             {
                 dlg->show();
-                dlg->activateWindow();
+                if (this->isActiveWindow())
+                    dlg->activateWindow();

```

Comment:
IMHO for accessibility reasons getting the focus automatically should not occurs at all in this case.